.yarnrc.yml

```
@@ -11,3 +11,9 @@ packageExtensions:
   "@vitest/browser@*":
     peerDependencies:
       vite: "^6"
+  "plugma@*":
+    dependencies:
+      nanoid: "*"
+  "@typescript-eslint/type-utils@*":
+    dependencies:
+      "@typescript-eslint/types": "*"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)